shared/team-building/container.tsx

```diff
@@ -136,6 +136,7 @@ const emptyObj = {}
 
 const mapStateToProps = (state: Container.TypedState, ownProps: OwnProps) => {
   const teamBuildingState = state[ownProps.namespace].teamBuilding
+  const teamBuildingSearchResults = teamBuildingState.teamBuildingSearchResults.toJS()
   const userResults = teamBuildingState.teamBuildingSearchResults.getIn([
```

Comment:
It looks like `userResults` is very similar to `teamBuildingSearchResults`, can we use that instead?

Reply:
Hmm, it looks like we could but I'm not sure whether it is a cleaner design. Using it would require having phone-search track the current phone number in redux so that this container can access the current phone number and pull out the result? Does that seem cleaner? To me it seems like that adds a bit more indirection than is necessary.